server/server.go

```diff
@@ -17,6 +17,7 @@ type Server struct {
 func New(svc *service.Service) *Server {
 	router := mux.NewRouter()
 	router.HandleFunc("/v0/data-deal/deals/{dealId}/data", svc.ValidateData).Methods("POST")
+	router.HandleFunc("/v0/data-deal/combined-key?deal_id={dealId}&data_hash={dataHash}}", svc.GetCombinedKey).Methods("GET")
```

Comment:
```suggestion
	router.HandleFunc("/v0/data-deal/secret-key?deal_id={dealId}&data_hash={dataHash}}", svc.GetCombinedKey).Methods("GET")
```
@medibloc/panacea-dev 
Let's not use the word `combined` when we interact with end users, since they don't need to know details about how secret keys are generated. Of course, we can use call it combined key internally.

Reply:
Also, let's alway go with hyphens instead of underscores.